src/commons/api/ca.rs

```diff
@@ -1503,16 +1490,11 @@ impl From<&IssuedCert> for ParentStatusCert {
 pub struct ParentStatus {
     last_exchange: Option<ParentExchange>,
     last_success: Option<Timestamp>,
-    next_exchange_before: Timestamp,
```

Comment:
Is this a breaking API change?

Reply:
yes, well - we never committed to this being stable so I think it's okay with the updated readme.